lib/std/fs.zig

```diff
@@ -2251,7 +2251,7 @@ pub fn selfExePath(out_buffer: []u8) SelfExePathError![]u8 {
                 const PATH = std.os.getenvZ("PATH") orelse return error.FileNotFound;
                 var path_it = mem.tokenize(PATH, &[_]u8{path.delimiter});
                 while (path_it.next()) |a_path| {
-                    var resolved_path_buf: [MAX_PATH_BYTES]u8 = undefined;
+                    var resolved_path_buf: [MAX_PATH_BYTES-1:0]u8 = undefined;
```

Comment:
I'm wondering, could this not be made into

```zig
var resolved_path_buf: [MAX_PATH_BYTES:0]u8 = undefined;
```

instead?

Reply:
in https://github.com/ziglang/zig/pull/6638#discussion_r502887556, I was told to use `MAX_PATH_BYTES-1:0`.

Reply:
Cool! I was just wondering why not `MAX_PATH_BYTES + 1`, but I guess it makes sense for it to be null-delimited, hence the null makes the last byte in `MAX_PATH_BYTES`. :-)